src/main/resources/application.properties

```diff
@@ -1 +1 @@
-server.port=8081
+server.port=8080
```

Comment:
The port was set to 8081 intentionally to prevent clashes with other applications that might load on port 8080 (in my case Oracle TNSLSNR Executable). Please keep it at 8081 while in development phase